eternalcore-core/src/main/java/com/eternalcode/core/feature/rtp/RTPService.java

```diff
@@ -0,0 +1,119 @@
+package com.eternalcode.core.feature.rtp;
+
+import com.eternalcode.core.configuration.implementation.PluginConfiguration;
+import org.bukkit.Location;
+import org.bukkit.Material;
+import org.bukkit.World;
+import org.bukkit.block.Block;
+import org.bukkit.entity.Player;
+
+import java.util.List;
+import java.util.Random;
+
+public class RTPService {
+
+    private static final List<Material> UNSAFE_BLOCKS = List.of(
+        Material.LAVA,
+        Material.WATER,
+        Material.CACTUS,
+        Material.FIRE,
+        Material.COBWEB,
+        Material.SWEET_BERRY_BUSH,
+        Material.MAGMA_BLOCK
+    );
```

Comment:
Można do configu wydzielić

Reply:
raczej to jest już internal impl niż do konfiguracji